coderd/dormancy/notifications.go

```diff
@@ -0,0 +1,43 @@
+package dormancy
```

Comment:
Since dormancy is an enterprise-only feature, you should consider moving this to the existing `dormancy` package in `enterprise/coderd/dormancy`.